CMakeLists.txt

```diff
@@ -63,10 +65,21 @@ function(trusted_firmware_build)
     set(TFM_PROFILE_ARG -DTFM_PROFILE=${TFM_BUILD_PROFILE})
   endif()
 
+  if(DEFINED TFM_PSA_TEST_SUITE)
+    set(PSA_TEST_ARG -DTEST_PSA_API=${TFM_PSA_TEST_SUITE})
```

Comment:
Based on this info:
```
# PSA_TEST_SUITE: A PSA test suite to add, choose one of
#                 STORAGE/CRYPTO/INITIAL_ATTESTATION
```
it seems there is only three valid values.

So I suggest to add an additional check, like:
```
set(VALID_PSA_TEST_SUITES STORAGE CRYPTO INITIAL_ATTESTATION)

# And then later:
  if(DEFINED TFM_PSA_TEST_SUITE)
    if(NOT TFM_PSA_TEST_SUITE IN_LIST VALID_PSA_TEST_SUITES)
      message(FATAL_ERROR "${TFM_PSA_TEST_SUITE} not valid, please use one of: ${VALID_PSA_TEST_SUITES}")
    endif()

   set(PSA_TEST_ARG -DTEST_PSA_API=${TFM_PSA_TEST_SUITE}) 
  endif()